packages/scale/index.d.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Get scale properties. It returns an object with:
+ * - name: the scale name
+ * - names: a list with all possible names (includes the current)
+ * - intervals: an array with the scale intervals
+ * - chroma:  scale croma (see pcset)
+ * - setnum: scale chroma number
+ *
+ * @function
+ * @param {String} name - the scale name (without tonic)
+ * @return {Object}
+ */
+export declare const props: (
+  name: string
+) => {
+  name: string;
+  names: string[];
+  intervals: string[];
+  chroma: string;
+  setnum: string;
+};
+/**
+ * Return the available scale names
+ *
+ * @function
+ * @param {boolean} aliases - true to include aliases
+ * @return {Array} the scale names
+ *
+ * @example
+ * Scale.names() // => ["maj7", ...]
+ */
+export declare const names: (aliases: boolean) => string[];
```

Comment:
Not documented, but is:
```suggestion
export declare const names: (aliases?: boolean) => string[];
```

And with more types:
```ts
export declare const names: (aliases: boolean) => ScaleName[];
```